blocks/search/search.css

```diff
@@ -0,0 +1,76 @@
+/* Search Block CSS */
```

Comment:
Note: We'll clean up some of the stylings to ensure that we're using the spacings and etc from the design specs.